apps/web/components/Kbar.tsx

```diff
@@ -106,31 +106,31 @@ export const KBarRoot = ({ children }: { children: React.ReactNode }) => {
       section: "Profile Settings",
       shortcut: ["p", "s"],
       keywords: "setting profile",
-      perform: () => router.push("/settings"),
+      perform: () => router.push("/settings/profile"),
```

Comment:
Directly going to `settings/profile` is working but if we put `settings` alone, it looks to access user property before it is accessible. This works as a hotfix 🤷